uint16 joystick_1_x
uint16 joystick_1_y
uint16 joystick_2_x
uint16 joystick_2_y
bool joystick_1_press
bool joystick_2_press